rest_framework_json_api/views.py

```diff
@@ -98,6 +100,54 @@ def get_queryset(self, *args, **kwargs):
         return qs
 
 
+class RelatedMixin(object):
```

Comment:
Is there any reason we shouldn't directly derive `ModelViewSet` from this mixin? This would be easier to use but have to think about whether it breaks backwards compatibility.

Reply:
I think so too so let's extend `ModelViewSet` and `ReadOnlyViewSet` with this mixin

Reply:
Done

Reply:
Great.